test/oli_web/live/user_live_test.exs

```diff
@@ -0,0 +1,121 @@
+defmodule OliWeb.UserLiveTest do
+  use ExUnit.Case
+  use OliWeb.ConnCase
+
+  import Phoenix.LiveViewTest
+  import Oli.Factory
+
+  @live_view_author_index_route Routes.live_path(OliWeb.Endpoint, OliWeb.Users.AuthorsView)
+
+  describe "user cannot access when is not logged in" do
+    test "redirects to new session when accessing the index view", %{conn: conn} do
+      {:error, {:redirect, %{to: "/authoring/session/new?request_path=%2Fadmin%2Fauthors"}}} =
+        live(conn, @live_view_author_index_route)
+    end
+  end
+
+  describe "user cannot access when is logged in as an author but is not a system admin" do
+    setup [:author_conn]
+
+    test "returns forbidden when accessing the index view", %{conn: conn} do
+      conn = get(conn, @live_view_author_index_route)
+
+      assert response(conn, 403)
+    end
+  end
+
+  describe "index" do
```

Comment:
We already add some tests for this route here: https://github.com/wyeworks/oli-torus/blob/master/test/oli_web/live/admin_live_test.exs#L493

Maybe we can update them there to reflect this changes as well?